lib/Core/Repository/Aggregate/Repository.php

```diff
@@ -64,9 +64,12 @@ public function setCurrentUser(UserReference $user)
         return $this->ezRepository->setCurrentUser($user);
     }
 
-    public function sudo(Closure $callback)
+    public function sudo(callable $callback, RepositoryInterface $outerRepository = null)
```

Comment:
Does changing `Closure` to `callable` makes this incompatible with previous versions of eZ kernel?

Reply:
No; previously the method was not in the interface and `callable` type hint will allow `Closure` instance. I updated the test setup to allow testing integration with different versions of `ezpublish-kernel`, everything works OK.

Reply:
👍